### PR TITLE
pushBack --> Set

### PR DESCRIPTION
**Pull request by DardoTheMaster**
Using set instead of pushBack, it will avoid added entries to the database to conflict with framework ones.

### CODE REVIEW

**Comment by BoGuu** [comment]
Thanks for reminding me to make them all pushBack!

People work around us, not the other way.

**Comment by dbenson24** [comment]
@BoGuu if the goal is to make them all pushBack why merge this now?

**Comment by DardoTheMaster** [comment]
@dbenson24 I think this is just a temp alternative... Just give boguu the time to handle the script using pushback..

**Comment by BoGuu** [comment]
Don't forget I'm not the one merging these. It serves no use though, stick your extra stuff after it, or change it yourselves to fit needs, instead of sacrificing performance.

**Comment by tkcjesse** [comment]
Temp alternative to what people being lazy? Seems awfully pointless.

**Comment by DardoTheMaster** [comment]
Lol you guys really are not the right people to handle a framework

**Comment by dbenson24** [comment]
@DardoTheMaster I was genuinely curious, I'm glad you made this PR. I think that the initialization code is confusing as it is, explicitly using set just makes the code clearer. I don't think the performance gains on pushBack are really worth it in this case

**Comment by BoGuu** [comment]
@DardoTheMaster 

I'm purely making the observation that people work and build around the Framework, we are not here to make sure everything is as straight forward as humanly possible. If people are unable to work out what pushBack is doing, and count, that is their problem, it doesn't take much effort for them to learn, or even read the BI wiki. If I was suggesting something that would make this code utterly unreadable, I would sympathize, but I'm not. I made a statement that adding your code after the previous takes no effort and has nice rewards, nobody is trying to be a dick.

However, if you feel you can do a much better job, nobody is stopping you.

@dbenson24 

The initialization process is ugly and hard to follow, I agree, but using pushBack is not the cause of that, following a massive train of executions with lack of explanation is. In no case should we say the performance gains are not worth it, given the fact that the code is equally as readable, it hasn't turned into a battle of obfuscation.
